### ISSUE TITLE
[MessageBar] No way to differentiate between expanded/collapsed message bar for screen readers

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#5717]
i) Aria label for the dropdown button available is fixed irrespective of its expand/collapse status
ii) "IsExpandCollapsePatternAvailable" property is "False" for the dropdown button.

Only overflowButtonAriaLabel property is available with no callback for expand/collapse. As a result, the consumer of component can not control screen reader behavior

#### Example:
 https://developer.microsoft.com/en-us/fabric#/components/messagebar

#### Component:
![image](https://user-images.githubusercontent.com/11937034/43389317-0d540f26-9409-11e8-8c69-4a5657c69030.png)
#### Accessibility properties of button:
![image](https://user-images.githubusercontent.com/11937034/43389335-1658d8c2-9409-11e8-9cbc-922eb15bc081.png)
#### Narrator behavior (narrator buddy):
![image](https://user-images.githubusercontent.com/11937034/43389889-78d031e8-940a-11e8-8eca-825160f1486f.png)
There are no announcements on activating/deactivating the overflow button

#### Expected behavior:
 
Name property value should be 'Expand' when the button is collapsed and 'Collapse' when the button is expanded for the dropdown button.
                                        OR
The button should have the descriptive name property value and 'IsExpandCollapsePatternAvailable' property should be 'true'

**Comment by natalieethell** [comment]
Hey @prmittal thanks for submitting this issue! @betrue-final-final and I discussed an approach, and I think it makes the most sense to add an aria-collapsed property. (I'm assuming that's related to the IsExpandCollapsePatternAvailable you're referring to, so correct me if I'm wrong).

**Comment by prmittal** [comment]
@natalieethell I see that the fix made was to add the property to message bar div and not the "See more" button. Would that not be the preferred fix?

**Comment by betrue-final-final** [comment]
You're suggesting changing the label of the button to "See more"? I can see that being a good change. "Overflow" is more of a UX professional term and not as accessible as "See more".

**Comment by prmittal** [comment]
No. I meant setting the aria-expanded property to the button instead of the full message bar div. The div is not actionable.

**Comment by KevinTCoughlin** [comment]
Thanks for raising awareness of this issue @prmittal 😃
